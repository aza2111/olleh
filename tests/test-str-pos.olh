//test-str-pos: string operations

String str = "a string";
str = str + 5 + 6; //should be "a string56";
print str;

str = "
i have 3
lines";

print str;

str = "asdf " + " asdf";
print str;

str = "adsf\nx";
print str;
str = "abcd";
print str;
str = "\0ooo";
print str;
