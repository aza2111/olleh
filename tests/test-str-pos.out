a string56

i have 3
lines
asdf  asdf
adsf\nx
abcd
\0ooo
